etc/systemd/system/modbot.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Strims.gg modbot
+After=network.target
+
+[Service]
+User=rustla
+Restart=always
+WorkingDirectory=/home/rustla/modbot
+ExecStart=/usr/bin/modbot
```

Comment:
Do we (go)install the modbot? probably just use the binary in home path for convenience...

also im not familiar with adding git files with an absolute path, need to check this out. (the web review interface seems to show that)

Reply:
yes go install
if you don't want that, we can change the path easily